parm/soca/berror/soca_ensrecenter.yaml

```diff
@@ -0,0 +1,102 @@
+# This yaml is for applying deterministic recentering increments to the ensemble members
+
+geometry:
+  mom6_input_nml: mom_input.nml
+  fields metadata: ./fields_metadata.yaml
+
+date: '{{ATM_WINDOW_BEGIN}}'
+
+layers variable: [hocn]
+
+increment variables: [tocn, socn, uocn, vocn, ssh, hocn, cicen, hicen, hsnon]
+
+set increment variables to zero: [ssh]
+
+vertical geometry:
+  date: '{{ATM_WINDOW_BEGIN}}'
+  basename: ./INPUT/
+  ocn_filename: MOM.res.nc
+  read_from_file: 3
+
+add recentering increment: false
+
+soca increments:  # Could also be states, but they are read as increments
+  number of increments: 4 $${ENS_SIZE}
+  pattern: '%mem%'
+  template:
+    date: '{{ATM_WINDOW_BEGIN}}'
+    basename: ./ens/
+    ocn_filename: 'ocean.%mem%.nc'
+    ice_filename: 'ice.%mem%.nc'
+    read_from_file: 3
+
+steric height:
+  linear variable changes:
+  - linear variable change name: BalanceSOCA  # Only the steric balance is applied
+
+#ensemble mean output:
+#  datadir: ./static_ens
+#  date: '{{ATM_WINDOW_BEGIN}}'
```

Comment:
I'm going to stop from here on out, but all of these should have spaces between the bracket and variable name per Jinja2 standard

Reply:
done